synapse/storage/databases/main/receipts.py

```diff
@@ -294,12 +294,16 @@ def f(txn):
                 sql = """
                     SELECT * FROM receipts_linearized WHERE
                     stream_id > ? AND stream_id <= ?
+                    ORDER BY stream_id DESC
+                    LIMIT 100
```

Comment:
Can you also change the comment at https://github.com/matrix-org/synapse/blob/c276bd996916adce899410b9c4c891892f51b992/synapse/handlers/receipts.py#L161 which I believe is now out of date.

Also putting something about the limit in this function's docstring would be helpful.

Reply:
Updated.